nml/actions/action2var_variables.py

```diff
@@ -779,30 +773,64 @@ def industry_cargotype(name, args, pos, info):
 }
 
 
-varact2vars[0x00] = varact2vars_trains
-varact2vars60x[0x00] = varact2vars60x_trains
-varact2vars[0x01] = varact2vars_roadvehs
-varact2vars60x[0x01] = varact2vars60x_roadvehs
-varact2vars[0x02] = varact2vars_ships
-varact2vars60x[0x02] = varact2vars60x_vehicles
-varact2vars[0x03] = varact2vars_aircraft
-varact2vars60x[0x03] = varact2vars60x_vehicles
-varact2vars[0x04] = varact2vars_stations
-varact2vars60x[0x04] = varact2vars60x_stations
-varact2vars[0x05] = varact2vars_canals
-varact2vars[0x07] = varact2vars_houses
-varact2vars60x[0x07] = varact2vars60x_houses
-varact2vars[0x09] = varact2vars_industrytiles
-varact2vars60x[0x09] = varact2vars60x_industrytiles
-varact2vars[0x0A] = varact2vars_industries
-varact2vars60x[0x0A] = varact2vars60x_industries
-varact2vars[0x0D] = varact2vars_airports
-varact2vars60x[0x0D] = varact2vars60x_airports
-varact2vars[0x0F] = varact2vars_objects
-varact2vars60x[0x0F] = varact2vars60x_objects
-varact2vars[0x10] = varact2vars_railtype
-varact2vars[0x11] = varact2vars_airporttiles
-varact2vars60x[0x11] = varact2vars60x_airporttiles
-varact2vars[0x12] = varact2vars_roadtype
-varact2vars[0x13] = varact2vars_tramtype
-varact2vars[0x14] = varact2vars_towns
+class VarAct2Scope:
+    def __init__(self, name, vars_normal, vars_60x, has_persistent_storage=False):
+        self.name = name
+        self.vars_normal = vars_normal
+        self.vars_60x = vars_60x
+        self.has_persistent_storage = has_persistent_storage
+
+
+class VarAct2Feature:
+    def __init__(self, self_scope, parent_scope):
+        self.self_scope = self_scope
+        self.parent_scope = parent_scope
+
+    def get_scope(self, var_range):
+        assert var_range in (0x89, 0x8A)
```

Comment:
asserts get removed in optimised builds (like Windows exe), and I think most others have been replaced with an explicit `raise AssertionError`. Not necessarily an issue, but something to consider?

Reply:
We removed all "assert False". If they were removed in release builds, they would open up new code paths.
A tool warned about that.
Regular asserts are fine.